<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPvdbRatingDialog</class>
 <widget class="QDialog" name="QtPvdbRatingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>De kentallen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ProjectVanDenBogaart.qrc">
    <normaloff>:/images/PicArrow14x14.png</normaloff>:/images/PicArrow14x14.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_student_name">
     <property name="text">
      <string>Student name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_assessor_name" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_assessor_name">
        <property name="text">
         <string>Assessor name: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Overzicht concepten</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_concepts">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Complexiteit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Concreetheid</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Specificiteit</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Verdeling van de voorbeelden/toelichting over de zes objecten van kennis</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_examples">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>235</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>324</width>
       <height>235</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <row>
      <property name="text">
       <string>Kennis van het beroepsdomein</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Kennis van organisaties</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Kennis van de sociale omgeving</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Kennis van de doelgroep</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Technisch instrumentele kennis</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Kennis van de eigen persoon</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Overige</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>%</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Gevonden waarden</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_values">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>144</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <row>
      <property name="text">
       <string>Complexiteit</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Concreetheid</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Specificiteit</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Rijkheid</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Experimentele waarde</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Geschatte waarde</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>X_e</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>X_s</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>C_e</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>C_s</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>S_e</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>R_e</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>-</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_bottom" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_save">
        <property name="text">
         <string>&amp;Sla assessment rapport op</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_print">
        <property name="text">
         <string>&amp;Print het assessment rapport</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ProjectVanDenBogaart.qrc"/>
 </resources>
 <connections/>
</ui>
